src/store/actions/loadStateActions.ts

```diff
@@ -0,0 +1,29 @@
+export const reducerDomains = {
+    AUTH: 'AUTH',
+    PROFILE: 'PROFILE',
+    TASKS: 'TASKS',
+    CALENDAR: 'CALENDAR',
+};
+
+export const loadStateActionTypes = {
+    LOADED: 'LOADED',
+    LOADING: 'LOADING',
+    ERROR: 'ERROR',
+};
```

Comment:
change to typescript enums
https://www.typescriptlang.org/docs/handbook/enums.html